src/scala/runtime/ScalaRunTime.scala

```diff
@@ -50,7 +50,13 @@ object ScalaRunTime {
   /** Return the class object representing an array with element class `clazz`.
     */
   def arrayClass(clazz: jClass[_]): jClass[_] = {
-    ??? // Dmitry: I want to see where this method is used to know how to fix it
+    // Dmitry: I want to see where this method is used to know how to fix it
+    // [Martin] Here's a stacktrace
+    // Exception in thread "main" scala.NotImplementedError: an implementation is missing
+    //  at scala.Predef$.$qmark$qmark$qmark(Predef.scala:225)
+    //  at scala.runtime.ScalaRunTime$.arrayClass(ScalaRunTime.scala:53)
+    //  at scala.Array$.fill(Array.scala:281)
```

Comment:
Which version of scala-library do you have? I do not see any usage of `arrayClass` in `Array.scala:281`.